### PR TITLE
Fix Docker issue

### PR DESCRIPTION
**Pull request by alejandro-kid**
I did three changes:
- Change th way of build the container.
- Change script to generate application.yml:
  > Explanation: The awk command iterates through each element of a list and executes its instructions, in each iteration it checks that the corresponding environment variable contains information. Since it is impossible to declare a valid environment variable for the system with the following structure 'Cenit::Admin:default_uri', it is decided to create it from one that is valid and eliminate the other environment variable declared in the file, to be provided by the system globally.
- It necesary when the container run, declare a variable called: ADMIN_UI.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -44,16 +44,9 @@ COPY Gemfile .
...
-COPY app app/
+COPY . .
+RUN chmod +x env.sh
 
 RUN yarn install --check-files
 
-COPY ./env.sh .
-RUN chmod +x env.sh
-
+CMD ["/bin/bash", "-c", "/var/www/cenit/env.sh; bundle exec unicorn_rails -c config/unicorn.rb"]
```
Comment (sanchojaf) [inline Dockerfile:52]: keep this file like was before and add this line at the end

File: env.sh [hunk 2]
```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-awk -F '=' '{ print $1 ": \"" (ENVIRON[$1] ? ENVIRON[$1] : $2) "\"" }' ./application.docker.yml >> ./config/application.yml
+awk -F '=' '{ print $1 ": \"" (ENVIRON["ADMIN_UI"] ? ENVIRON["ADMIN_UI"] : $2) "\"" }' ./application.docker.yml >> ./config/application.yml
```
Comment (sanchojaf) [inline env.sh:2]: remove ./application.docker.yml and just add the end of ./config/application.yml ENVIRON["ADMIN_UI"]

File: docker-compose.yml [hunk 3]
```diff
@@ -17,12 +17,12 @@ services:
       - MONGODB_URI=mongodb://mongo_server/cenit
       - REDIS_HOST=redis
       - HOMEPAGE=http://localhost:3000
+      - ADMIN_UI=http://localhost:3002
```
Comment (sanchojaf) [inline docker-compose.yml:20]: good

Comment (sanchojaf) [reply docker-compose.yml:20]: I suggest naming it CENIT_UI

**Review by sanchojaf — CHANGES_REQUESTED** [review]

File: application.docker.yml [hunk 4]
```diff
@@ -1,2 +1 @@
-HOMEPAGE=http://localhost:3000
 'Cenit::Admin:default_uri'=http://localhost:3002
```
Comment (sanchojaf) [inline application.docker.yml:1]: remove this file

**Comment by alejandro-kid** [comment]
- Delete application.docker.yml: Now the config/application.yml will be create inline using an script when the container run. For this purpose need declared two environment variables HOMEPAGE and CENIT_UI.
  > Note: The main conclusion of all my test is that in application.yml need to have both env variables (HOMEPAGE and Cenit::Admin:default_uri), even if HOMEPAGE is not in it, the platform will not redirect correctly.
- Update docker-compose.yml: Here I deleted the configuration of NGINX. Docker compose is not for productions, so dont have sense configure NGINX, i remove this element to do the file more simple.
- Enable build container in developer branch and redefine tags of container: I think that the best way to resolve a bug is knowin where is it. For this reason I configured a system of tags, the tags of master branch will be latest and a short sha (the first 7 numbers), because with the current tag when a new container is pushed override the previusly. And in developer branch the container will have a tag as following: develop-${SHA}

**Review by sanchojaf — APPROVED** [review]

**Pull request merged**